config/src/config/consensus_config.rs

```diff
@@ -57,6 +57,8 @@ pub struct ConsensusConfig {
     // must match one of the CHAIN_HEALTH_WINDOW_SIZES values.
     pub window_for_chain_health: usize,
     pub chain_health_backoff: Vec<ChainHealthBackoffValues>,
+    // length of inbound queue of messages
+    pub max_receiving_messages: usize,
```

Comment:
Nice! Can we name them `max_network_channel_size`, so that they match the one already defined by state sync? https://github.com/aptos-labs/aptos-core/blob/7088dd699018e5e2ffa4f85b3336df69d10941b3/config/src/config/state_sync_config.rs#L116